panoptes_aggregation/routes.py

```diff
@@ -69,6 +70,16 @@ def make_application():
                         static_url_path='',
                         static_folder='../docs/build/html')
     application.json_encoder = MyEncoder
+    CORS(
+        application,
+        origins=[
+            r'^https?:\/\/([a-z0-9-.]+zooniverse.org)',
+            r'^https?:\/\/(127\.0\.0\.1|localhost|[a-z0-9-]+\.local)(:\d+)?$'
```

Comment:
i'd prefer we use `local.zooniverse.org` for local dev on production system. If this system can't leak any private data then it's probably ok for localhost as well.

Reply:
@wgranger do you have `local.zooniverse.org` set up on you system (e.g. https://stackoverflow.com/c/zooniverse/questions/109) if so I will adjust the first regex to work when a port is put on the end.